Sources/RInAppMessaging/Utilities/AnalyticsTracker.swift

```diff
@@ -0,0 +1,16 @@
+import class UIKit.UIDevice
+
+#if SWIFT_PACKAGE
+import class RSDKUtilsMain.AnalyticsBroadcaster
+#else
+import class RSDKUtils.AnalyticsBroadcaster
+#endif
+
+internal struct AnalyticsTracker {
+    static func sendEventName(_ name: Constants.RAnalytics, dataObject: [String: Any]? = nil) {
+        var eventData = dataObject ?? [:]
+        eventData[Constants.RAnalytics.Keys.deviceID] = UIDevice.current.identifierForVendor?.uuidString
```

Comment:
I think it would be better if device Id came from one source for the whole SDK code. To have a helper class which returns `UIDevice.current.identifierForVendor?.uuidString` as device ID and can be referenced here AND also in request classes

Reply:
Done